### PR TITLE
Format sections list

### PR DESCRIPTION
**Pull request by kevinjihwanlee**

### CODE REVIEW

**Comment by kevinjihwanlee** [comment]
[Format sections list](https://trello.com/c/GXnwmdtC/32-format-sections-list)

**Comment by kevinjihwanlee** [comment]
[Format sections list](https://trello.com/c/GXnwmdtC/32-format-sections-list)

File: src/components/Sidebar/SectionSelection.js [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+        {sections.map(section => (
+          <ListItem
+            key={section.id}
+            button
+          >
+            <ListItemText>
+              <Typography variant="h1" className={classes.sectionTitle}>{`Section ${section.section}`}</Typography>
+              <Typography>{`${section.class_mtg_info[0].meet_t}`}</Typography>
+              <Typography>{`${section.class_mtg_info[0].meet_l}`}</Typography>
+              <Typography>{section.instructor.map((t, i) => (`${t}${i !== section.instructor.length - 1 ? ', ' : ''}`))}</Typography>
```
Comment (Joonpark13) [inline src/components/Sidebar/SectionSelection.js]: Let's try to avoid use of single letter variables names. We can definitely be descriptive AND concise.

Comment (Joonpark13) [reply src/components/Sidebar/SectionSelection.js]: Also I feel like the length of this line should throw a linting error?

Comment (Joonpark13) [reply src/components/Sidebar/SectionSelection.js]: Tests are also failing because we're missing coverage on this line

**Review by Joonpark13 — CHANGES_REQUESTED** [review]

**Comment by Joonpark13** [comment]
Not a big deal this time since I pulled down and looked, but a reminder that with PRs that contain visual changes, screenshots are much appreciated!

**Comment by Joonpark13** [comment]
Looks good visually, but there's a console error when I click on a course to show sections. Can you address that?

**Comment by kevinjihwanlee** [comment]
screenshot 2018-11-04 00 31 37

**Review by Joonpark13 — APPROVED** [review]

**Pull request merged**